diff --git a/node_modules/tronweb/lib/esm/index.d.ts b/node_modules/tronweb/lib/esm/index.d.ts
index 20fc258..208db60 100644
--- a/node_modules/tronweb/lib/esm/index.d.ts
+++ b/node_modules/tronweb/lib/esm/index.d.ts
@@ -18,6 +18,25 @@ import { TronWeb } from './tronweb.js';
 export { TronWeb };
 import * as Types from './types/index.js';
 export { Types };
+
+export type {
+    AbiFragment,
+    Account,
+    BroadcastReturn,
+    ContractAbiInterface,
+    ContractParamter,
+    CreateSmartContractTransaction,
+    DefaultAddress,
+    GetTransactionResponse,
+    SignedTransaction,
+    Transaction,
+    TransactionInfo,
+    TriggerSmartContract,
+    TronWebOptions,
+} from './types';
+
+export type { EventResponse } from './lib/event.js';
+
 declare const _default: {
     utils: {
         address: typeof import("./utils/address.js");
